mitiq/calibration/settings.py

```diff
@@ -220,21 +221,33 @@ def to_dict(self) -> Dict[str, Any]:
         return summary
 
     def print_line(self, performance: str, circuit_type: str) -> None:
-        summary = self.to_dict()
-        str_scale_factors = str(summary["scale_factors"])[1:-1]
-        row = (
-            performance,
-            circuit_type,
-            self.technique.name,
-            summary["factory"][:-7],
-            str_scale_factors,
-            summary["scale_method"],
-        )
-        print(
-            "| {:^10} | {:^7} | {:^6} | {:<13} | {:<13} | {:<20} |".format(
-                *row
+        if self.technique is MitigationTechnique.ZNE:
+            summary = self.to_dict()
+            str_scale_factors = str(summary["scale_factors"])[1:-1]
+            row = (
+                performance,
+                circuit_type,
+                self.technique.name,
+                summary["factory"][:-7],
+                str_scale_factors,
+                summary["scale_method"],
+            )
+            print(
+                "| {:^10} | {:^7} | {:^6} | {:<13} | {:<13} | {:<20} |".format(
+                    *row
+                )
+            )
+        elif self.technique is MitigationTechnique.PEC:
+            summary = self.to_dict()
+            row = (
+                performance,
+                circuit_type,
+                self.technique.name,
+                summary["representation_function"],
+                str(summary["operations"]),
+                str(summary["noise_level"]),
             )
-        )
+            print(row)
```

Comment:
When I run this with the `light_pec_settings` object defined in the tests I get
```
| performance | circuit | method | representation function | operations to mitigate | noise level |  qubit dependent |
| ----------- | ------- | ------ | ----------------------- | ---------------------- | ----------- | --------------- | 
('✅', 'mirror', 'PEC', <function represent_operation_with_local_depolarizing_noise at 0x2880f0940>, '[cirq.Circuit([\n    cirq.Moment(\n        cirq.CNOT(cirq.LineQubit(0), cirq.LineQubit(1)),\n    ),\n]), cirq.Circuit([\n    cirq.Moment(\n        cirq.CZ(cirq.LineQubit(0), cirq.LineQubit(1)),\n    ),\n])]', '0.001')
('❌', 'mirror', 'PEC', <function represent_operation_with_local_depolarizing_noise at 0x2880f0940>, '[cirq.Circuit([\n    cirq.Moment(\n        cirq.CNOT(cirq.LineQubit(0), cirq.LineQubit(1)),\n    ),\n]), cirq.Circuit([\n    cirq.Moment(\n        cirq.CZ(cirq.LineQubit(0), cirq.LineQubit(1)),\n    ),\n])]', '0.001')
```

I think we want to use the [`format`](https://docs.python.org/3/library/stdtypes.html#str.format) function like we do with ZNE above. We'll also want to make sure to use the function name so we don't print extraneous info like the memory address. Similarly, do we want to print the operations? Or is there a shorter way to represent them? I don't think putting them in a table like this is very readable.

Reply:
Good catch, thanks for pointing this out! 
I fixed the function name printing. 
As for the operations, I think users would want to track which operations are represented in the PEC calibration runs, even if that causes the table to be a bit bulky, but now the info is printed more compactly as operations (as intended).